docs/user/dashboard/lens-end-to-end/lens-end-to-end.asciidoc

```diff
@@ -0,0 +1,506 @@
+[float]
```

Comment:
This documentation is really specific and includes helpful screenshots at each step, and I think we can remove some of the duplicate information as a result.